packages/shared/styles/components/organisms/SfBottomNavigation.scss

```diff
@@ -4,55 +4,43 @@
   position: fixed;
   bottom: 0;
   left: 0;
-  z-index: var(--bottom-navigation-z-index, 1);
+  z-index: 1;
   display: flex;
   justify-content: space-around;
   align-items: flex-end;
   width: 100%;
-  height: var(--bottom-navigation-height, 3.75rem);
-  padding: var(--bottom-navigation-padding, 0 var(--spacer-sm));
-  background: var(--bottom-navigation-background, var(--c-white));
-  box-shadow: var(
-    --bottom-navigation-box-shadow,
-    0px -2px 10px rgba(var(--_c-gray-secondary-base), 0.15)
-  );
+  height: 3.75rem;
+  background: var(--c-white);
+  box-shadow: 0px -2px 10px rgba(var(--_c-gray-secondary-base), 0.15);
 }
 .sf-bottom-navigation-item {
   --icon-color: var(--c-link);
   background: transparent;
   border: 0;
-  top: var(--bottom-navigation-item-top);
-  display: flex;
+  flex: 1;
+  display: inline-flex;
   flex-direction: column;
   justify-content: center;
   align-items: center;
-  align-self: var(--bottom-navigation-item-align-self);
-  margin: var(--bottom-navigation-item-margin);
-  padding: var(
-    --bottom-navigation-item-padding,
-    var(--spacer-xs) 0 var(--spacer-xs) 0
-  );
-  color: var(--bottom-navigation-item-color, var(--c-link));
-  @include font(
-    --bottom-navigation-item-font,
-    var(--font-weight--light),
-    var(--font-size--xs),
-    1,
-    var(--font-family--primary)
-  );
+  padding: var(--spacer-xs) 0 var(--spacer-xs) 0;
+  color: var(--c-link);
+  font-weight: var(--font-weight--light);
+  font-size: var(--font-size--xs);
+  font-family: var(--font-family--primary);
```

Comment:
not directly related to this pr, but is there a reason to use `--spacer-xs` and `--font-size--xs`? why not `--font-size-xs`, `--font-family-primary`, `--font-weight-light` (...)? i don't think we should follow bem conventions on css vars :thinking:

Reply:
@leomp12 TBH no, I would prefer not to use --font-size--xs and --spacer-xs. But that should be in a separate PR where we rename all the global CSS variables. These are the current global defined one :).